### PR TITLE
Integration test on staking process with real BLS signed values

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added integration test that proves that a staking transaction with real BLS signature will work. Test data will be provided from the frontend testing process.

### CODE REVIEW

File: integrationTests/testProcessorNode.go [hunk 1]
```diff
@@ -74,6 +76,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go/testscommon"
 	"github.com/ElrondNetwork/elrond-go/update"
 	"github.com/ElrondNetwork/elrond-go/vm"
+	process2 "github.com/ElrondNetwork/elrond-go/vm/process"
```
Comment (sasurobert) [inline integrationTests/testProcessorNode.go:79]: change process2

File: integrationTests/testInitializer.go [hunk 2]
```diff
@@ -1358,6 +1385,38 @@ func CreateAndSendTransaction(
 	node.OwnAccount.Nonce++
 }
 
+// CreateAndSendTransactionFromWallet will generate a transaction with provided parameters, sign it with the provided
+// wallet's tx sign private key and send it on the transaction topic
+func CreateAndSendTransactionFromWallet(
```
Comment (sasurobert) [inline integrationTests/testInitializer.go:1390]: I think there is a function which does the same already. PlayerSendsTransactoin ?

**Review by sasurobert — APPROVED** [review]

**Review by sasurobert — CHANGES_REQUESTED** [review]
small stuff

File: integrationTests/testInitializer.go (hunk 2 above)
Comment (iulianpascalau) [reply integrationTests/testInitializer.go:1390]: PlayerSendsTransactoin is sent from the node's address. Wanted to used that but couldn't

**Review by sasurobert — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
👍

**Pull request merged**